src/main/java/it/gov/pagopa/merchant/controller/MerchantController.java

```diff
@@ -29,4 +33,7 @@ ResponseEntity<MerchantDetailDTO> getMerchantDetail(
             @PathVariable("initiativeId") String initiativeId,
             @PathVariable("merchantId") String merchantId
     );
+    @GetMapping("/initiatives")
+    @ResponseStatus(code = HttpStatus.OK)
+    List<InitiativeDTO> getMerchantInitiativeList(@RequestHeader("x-merchant-id") String merchantId);
```

Comment:
Is there a misalignment regarding the merchantId parameters form?

The above API suppose it to be a PathVariable, here it's supposed to be a RequestHeader: are both APIs invoked by the same client, possibly on the same useCase?

As Hint: when invoked by Merchant Portal, the header x-merchant-id will be added by the APIm extracting it from the JWT.
When invoked by Welfare Portal it cannot be an header.

If the answer is: the above API are intended for the Welfare Portal and this for Merchant Portal, it could be usefull to separate them into different Controller classes